DEVELOPMENT.md

```diff
@@ -0,0 +1,106 @@
+# Developing
+
+## Getting started
+
+Many of the instructions here replicate what's in the [tektoncd/pipeline development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md), with a couple of caveats currently.
+
+1. This project currently does not support being built with `ko`
+2. This project has not yet been tested with GKE
+3. The instructions here pertain to building the backend that lets us interact with Tekton resources. Frontend instructions are to follow.
+
+We would love to accomplish these tasks and to update this document, contributions welcome!
+
+### Requirements
+
+You must install these tools:
+
+1. [`go`](https://golang.org/doc/install): The language Tekton Dashboard is
+   built in
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+1. [`dep`](https://github.com/golang/dep): For managing external Go
+   dependencies. - Please Install dep v0.5.0 or greater.
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
+   interacting with your kube cluster
```

Comment:
Maybe we could mention a 'safe' kubectl version here as we know there's a problem with creds creation in some versions